src/routes/+layout.svelte

```diff
@@ -1,9 +1,11 @@
 <script lang="ts">
+	import navBar from '../lib/componenets/navBar.svelte'
```

Comment:
look at how I use my Input.svelte and where I import and export and custom component start with capital letter. Naming convention is PascalCase